Signal/src/ViewControllers/ConversationView/Cells/ConversationMediaView.swift

```diff
@@ -117,7 +174,10 @@ public class ConversationMediaView: UIView {
         stillImageView.layer.minificationFilter = kCAFilterTrilinear
         stillImageView.layer.magnificationFilter = kCAFilterTrilinear
         stillImageView.backgroundColor = Theme.offBackgroundColor
-        addMediaSubview(stillImageView)
+        addSubview(stillImageView)
+        stillImageView.autoPinEdgesToSuperviewEdges()
+        addUploadProgressIfNecessary(stillImageView) { (_) in
```

Comment:
TIOLI: Maybe this callback handler should be optional, since we're not always doing anything with it?

Reply:
Done.